torch/_inductor/lowering.py

```diff
@@ -4321,20 +4321,16 @@ def max_pool2d_checks(
     return kernel_size, stride, padding, dilation, use_fallback
 
 
-@register_lowering(prims._low_memory_max_pool2d_with_offsets, type_promotion_kind=None)
-def _low_memory_max_pool2d_with_offsets(
+def _max_pool2d_with_offsets(
     x,
     kernel_size,
     stride,
     padding,
     dilation,
     ceil_mode=False,
+    assert_fallback=None,
```

Comment:
do we still need this arg ?

Reply:
No, I removed it.

Reply:
We shouldn't need it anymore. It is only consumer is the `max_pool2d_checks` fn, and the call to that has been removed.